src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -388,6 +393,18 @@ public function newLine($count = 1)
     }
 
     /**
+     * @param resource $stream
+     */
+    public function setInputStream($stream)
+    {
+        if (!is_resource($stream)) {
+            throw new \InvalidArgumentException(sprintf('The stream must be a valid resource, %s given'), gettype($stream));
```

Comment:
Should be `Symfony\Component\Console\Exception\InvalidArgumentException` to conform `Symfony\Component\Console\Helper\QuestionHelper`